tools/example-module/generate-example-module.ts

```diff
@@ -60,9 +60,18 @@ function inlineExampleModuleTemplate(parsedData: AnalyzedExamples): string {
     return result;
   }, {} as any);
 
-  return fs
-    .readFileSync(require.resolve('./example-module.template'), 'utf8')
-    .replace(/\${exampleComponents}/g, JSON.stringify(exampleComponents, null, 2));
+  let loadText = '\nexport async function loadExample(id: string): Promise<any> {\n';
```

Comment:
This might be easier to read using an array:

```typescript
const loadText = [
  `export async function loadExample(id: string): Promise<any> {`,
  `  switch (id) {`,
  exampleMetadata.map(e => `  case '${e.id}':\nreturn import('@angular/components-examples/${e.module.packagePath}');`),
  `    default:\nreturn undefined;`,
  `  }`
].join('\n');

```